src/dialogs.gs

```diff
@@ -163,14 +162,76 @@ function dialogTimesheet() {
 
 /* Dialog: Worklog - END */
 
+/* Dialog: Update Fields in Jira Issues from Spreadsheet */
```

Comment:
Functions called from the menu and some helper functions. They are specific to the feature and should have better comments which I will do early next week. For now I do not see that they can have any affect on the rest of the functionality.